rose/server/player.py

```diff
@@ -27,9 +27,8 @@ def reset(self):
         self.score = 0
 
     def in_lane(self):
-        min_x = self.lane * config.cells_per_player
-        next_x = min_x + config.cells_per_player
-        return min_x <= self.x < next_x
+        current_lane = self.x // config.cells_per_player
```

Comment:
Not very clear why `x // cells_per_player` works but it is shorter so why not. To make sure we don't get this wrong, adding couple of trivial tests can be nice.

Reply:
thanks:
added some tests